api/src/accounts.rs

```diff
@@ -143,7 +154,12 @@ impl Account {
 
     // These functions map directly to endpoint functions.
 
+    /// Retrieves the [`AccountData`] for the associated account
```

Comment:
Do the square brackets do anything, or is this just a style choice?

Reply:
It does a reference in rust docs, but it might not show up great in the generated spec?